theme/tests/bootstrap.php

```diff
@@ -26,7 +26,7 @@
 global $_theme_files;
 $_theme_files = [];
 
-$_theme_root = realpath( __DIR__ . '/..' );
+$_theme_root = 'material-design-google';
```

Comment:
Here (in `bin/php-tests.sh`) we are setting soft link from theme to installed WP:

Which requires theme root to be hard-coded as otherwise it will come as `theme`.

https://github.com/material-components/material-design-for-wordpress/pull/320/files#diff-e8ed461a365df1059241c066220cfea782c37b68a047fe8ad37910a870a91f8bR208